src/test/java/chat/task/EventTest.java

```diff
@@ -0,0 +1,321 @@
+package chat.task;
+
+import chat.ChatException;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Unit testing class for functions in Event class
+ */
+
+public class EventTest {
+
+    private static String formatStr = "Please input with this format:\n" +
+            "event [name] /at [end date/time]-[start date/time]\n" +
+            "* start and end date/time should be written as dd/MM/yyyy HH:mm\n" +
+            "* i.e. 2019/03/19 22:00-2019/03/19 23:00\n" +
+            "* i.e. 2019/03/19 22:00 - 2019/03/19 23:00";
+
+    @Test
+    public void createEvent_wrongInstruction_chatException() {
+        String errorMessage = "wrong instruction for event\n" + formatStr;
+        try {
+            Event.createEvent("todo");
+            fail(); // the test should not reach this line
+        } catch (ChatException e) {
+            assertEquals(errorMessage, e.getMessage());
+        }
+    }
+
+    @Test
+    public void createEvent_onlyEvent_chatException() {
+        String errorMessage = "event name, start and end date/time missing\n" + formatStr;
+        try {
+            Event.createEvent("event");
+            fail();
+        } catch (ChatException e) {
+            assertEquals(errorMessage, e.getMessage());
+        }
+    }
+
+    @Test
+    public void createEvent_eventOneSpace_chatException() {
+        String errorMessage = "event name, start and end date/time missing\n" + formatStr;
+        try {
+            Event.createEvent("event");
+            fail();
+        } catch (ChatException e) {
+            assertEquals(errorMessage, e.getMessage());
+        }
+    }
+
+    @Test
+    public void createEvent_eventManySpaces_chatException() {
+        String errorMessage = "event name, start and end date/time missing\n" + formatStr;
+        try {
+            Event.createEvent("event       ");
+            fail();
+        } catch (ChatException e) {
+            assertEquals(errorMessage, e.getMessage());
+        }
+    }
+
+    @Test
+    public void createEvent_noSpaceBetweenEventAndDescription_chatException() {
```

Comment:
hmm isnt the standard to capitalize the first words, eg. CreateEvent_NoSpace..._ChatException

Reply:
Maybe you have remembered wrongly because I found this on the coding standard guide?

![image](https://user-images.githubusercontent.com/66005036/106916372-0ee01100-6742-11eb-9648-caaaa51da387.png)